apps/e2e/cypress/e2e/FAPs.cy.ts

```diff
@@ -429,7 +443,7 @@ context('Fap reviews tests', () => {
       cy.get('[data-cy="fap-assignments-table"] thead').contains('Deviation');
     });
 
-    it('Officer should be able to assign Fap member to proposal in existing Fap', () => {
+    it('only Officer should be able to assign Fap member to proposal in existing Fap', () => {
```

Comment:
Do we really need this `only` keyword here?

Reply:
No, its a mistake, I will fix it. Thanks for noticing.